combustion-android-ble/src/main/java/inc/combustion/framework/service/FoodSafeData.kt

```diff
@@ -226,22 +236,28 @@ sealed class FoodSafeData {
                 get() {
                     return when (this) {
                         Default -> 0u
-                        Beef -> 1u
-                        BeefGround -> 2u
-                        Chicken -> 3u
-                        ChickenGround -> 4u
-                        Pork -> 5u
-                        PorkGround -> 6u
-                        Ham -> 7u
-                        HamGround -> 8u
-                        Turkey -> 9u
-                        TurkeyGround -> 10u
-                        Lamb -> 11u
-                        LambGround -> 12u
-                        FishAndShellfish -> 13u
-                        FishAndShellfishGround -> 14u
+                        Meats -> 1u
+                        MeatsGround -> 2u
+                        DeprecatedChicken -> 3u
+                        PoultryGround -> 4u
+                        DepracatedPork -> 5u
+                        DeprecatedPorkGround -> 6u
+                        DeprecatedHam -> 7u
+                        DeprecatedHamGround -> 8u
+                        DeprecatedTurkey -> 9u
+                        DeprecatedTurkeyGround -> 10u
+                        DeprecatedLamb -> 11u
+                        DeprecatedLambGround -> 12u
+                        Seafood -> 13u
+                        SeafoodGround -> 14u
                         DairyMilkLessThan10PercentFat -> 15u
-                        Game -> 16u
+                        Other -> 16u
```

Comment:
I'm a little sad that we lost the context of `Game` coming from a guy who was cooking kangaroo meat. :)

Reply:
lol, kangaroo does not look very appetizing - I imagine rat taste